src/traces/contour/find_all_paths.js

```diff
@@ -83,14 +83,16 @@ function makePath(pi, loc, edgeflag, xtol, ytol) {
         pts.push(getInterpPx(pi, loc, marchStep));
         loc[0] += marchStep[0];
         loc[1] += marchStep[1];
+        locStr = loc.join(',');
 
         // don't include the same point multiple times
         if(equalPts(pts[pts.length - 1], pts[pts.length - 2], xtol, ytol)) pts.pop();
-        locStr = loc.join(',');
```

Comment:
This is not related to this PR. But I was wondering an error may happen in case of trying to access a negative element if/when `pts.length - 2 < 0`?

Reply:
That should never happen. `pts` has `length=1` before entering this loop and there's a `pts.push` a few lines above this statement.

Reply:
@etpinard Thanks for checking that.